### PR TITLE
Add Ayu, Ayu-Mirage & Ayu-Dark

### PR DESCRIPTION
**Pull request by jjsearle**
Colours are a bit different from 1.6 from looking at the old files but the new colours have been grabbed from here

https://raw.githubusercontent.com/ayu-theme/ayu-colors/master/colors.svg

### Ayu
ayu

### Ayu Mirage
ayu-mirage

### Ayu Dark
ayu-dark

### CODE REVIEW

**Review by kgilmer — APPROVED** [review]
These look fantastic @jjsearle !  I'm happy to merge this as is.  There a few things we could do here now or perhaps in a subsequent change.

Regarding style, most of [nerdfonts](https://www.nerdfonts.com/) are packaged for Regolith (notable exception is the Noto font, it's too big).  We could give each look a more distinctive look with [different fonts](https://www.nerdfonts.com/font-downloads).  Or, if you think SourceCodePro is best I won't argue with you.

Feel free to merge or look into other fonts, your call :sun_with_face:

**Comment by jjsearle** [comment]
Hey Ken - I have switched these three over to the 'Hack' font, if you are happy with this and don't want me to make any further changes please proceed with the merge. Thanks.

File: usr/share/regolith-look/ayu-dark/ilia.css
Comment (kgilmer) [inline usr/share/regolith-look/ayu-dark/ilia.css:17]: This needs to be updated to use new font spec

File: usr/share/regolith-look/ayu-mirage/ilia.css
Comment (kgilmer) [inline usr/share/regolith-look/ayu-mirage/ilia.css:17]: ditto

File: usr/share/regolith-look/ayu/ilia.css
Comment (kgilmer) [inline usr/share/regolith-look/ayu/ilia.css:17]: same here

**Review by kgilmer — CHANGES_REQUESTED** [review]
The ilia CSS needs the correct font spec, otherwise LGTM

File: usr/share/regolith-look/ayu/ilia.css
Comment (jjsearle) [reply usr/share/regolith-look/ayu/ilia.css]: done

**Comment by jjsearle** [comment]
I have corrected the ilya.css to use the new font

**Review by kgilmer — APPROVED** [review]
LGTM

**Pull request merged**
